### PR TITLE
Add `digiBibSubjectFacet` #1617

### LINKED ISSUE #1617: Make more `labels` working as aggregations to provide facets
Lobid facets use IDs. DigiBib uses labels. Thus we need labels to be typed as `keyword` to  enable ES' aggregations.

We should start with:
- [x]  `language` and`medium`
and later also:
- [x] `subject`

@blackwinter noted that it woul be nice to have ISIL as a field on its own.

@dr0i expects not a huge decline in overall ETL processing time. ES-cluster should be fine doing the extra work.

### PR DESCRIPTION
**Pull request by TobiasNx**
I suggest this version.

Resolves #1617.

### CODE REVIEW

File: src/main/resources/alma/fix/subjects.fix [hunk 1]
```diff
@@ -542,3 +542,14 @@ do list(path:"spatial[]", "var":"$i")
   replace_all("$i.focus.geo.lon","^Point\\((.*) .*\\)","$1")
   replace_all("$i.focus.geo.lat","^Point\\(.* (.*)\\)","$1")
 end
+
+set_array("digiBibSubjectFacet[]")
+do list(path:"subject[]","var":"$i")
+  if any_equal("$i.type","Concept")
```
Comment (blackwinter) [inline src/main/resources/alma/fix/subjects.fix:548]: Shouldn't this be `$i.type[]`?

Looking at test case `990197293880206441.json`, it should also have "Computer science", "Software engineering" and "Operating systems (Computers)" in the new field.

**Review by blackwinter — COMMENTED** [review]
Fine by me :) Thank you!

But I just noticed that there's also `type=Keyword`. Would you mind adding those as well?

File: src/main/resources/alma/fix/subjects.fix (hunk 1 above)
Comment (TobiasNx) [reply src/main/resources/alma/fix/subjects.fix]: You are right

**Comment by TobiasNx** [comment]
Like this?

**Review by blackwinter — APPROVED** [review]
Great, thanks!

Except that local (IZ) fields should not be included at all :( E.g. "FB 03" in `990156027740206441.json`. Will create a new issue for this (see #1687).

**Comment by TobiasNx** [comment]
@acka47 are you okay 1) with the name of this element 2) should we set this element public or exclude this from the public index

**Comment by dr0i** [comment]
Note: we once (when?) agreed to overtly serve the data as it is - with the advantage that useres can see what fieled exists and to implicitly know (well, most of the times) what queries can be build to get their stuff done.

**Review by acka47 — CHANGES_REQUESTED** [review]
See comment

**Comment by TobiasNx** [comment]
We use subjectslabels now

**Pull request merged**